models/cartDao.js

```diff
@@ -0,0 +1,41 @@
+const dataSource = require('./dataSource');
+
+const queryCartItems = async (userId) => {
+  try {
+    const data = await dataSource.query(
+      `
+      SELECT
+  carts.product_id AS productId,
```

Comment:
데이터를 줄 때는 id를 같이 주는 것이 좋습니다.
```suggestion
      SELECT
  carts.id AS cartId,
  carts.product_id AS productId,
```